src/test/scala/io/delta/sql/DeltaSparkSessionExtensionSuite.scala

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 Databricks, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.delta.sql
+
+import java.nio.file.Files
+
+import org.apache.spark.SparkFunSuite
+import org.apache.spark.sql.SparkSession
+
+class DeltaSparkSessionExtensionSuite extends SparkFunSuite {
+
+  private def verifyDeltaSQLParserIsActivated(spark: SparkSession): Unit = {
+    val input = Files.createTempDirectory("DeltaSparkSessionExtensionSuite").toFile.getCanonicalPath
```

Comment:
we should create a temporary directory with `org.apache.spark.util.Utils.createTempDir()`, so that spark will automatically delete it by call shutdown hook when this test failed.

Reply:
Good catch. We cannot call `org.apache.spark.util.Utils.createTempDir` here as that's `private[spark]`. But I can manually delete the directory. Will fix it.